improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -718,8 +651,7 @@ def test_basic(self):
         self.assertArrayEqual(result.attributes['weather_code'], self.wxcode)
         self.assertEqual(result.attributes['weather_code_meaning'],
                          self.wxmeaning)
-        self.assertArrayEqual(result.data,
-                              self.no_lightning_wxcode)
+        self.assertArrayEqual(result.data, self.expected_wxcode)
```

Comment:
Should we be using np.testing now?

Reply:
We've not moved away from IrisTest yet as far as I am aware (I will ask around). We are being more careful with the "approximately equal to" stuff from IrisTest due to possibly inappropriate defaults, but the tests in here are exact equality, so I think they are fine.

Reply:
This is why I asked. I know it is being used, and I'm not sure what we should be using. It all works so I think this is fine for me 👍